gradle/jacoco.gradle

```diff
@@ -1,7 +1,7 @@
 ext {
     AggregatedCoverageThresholds = [
-        "INSTRUCTION": 0.89,
-        "CLASS": 56
+        "INSTRUCTION": 0.8,
```

Comment:
```suggestion
        "INSTRUCTION": 0.89,
```

Reply:
see the screenshots from the comment. There, I intentionally made the pipelines fail to get these values. There, `0.8` is mentioned as the current threshold. As stated in the PR description, the aggregated thresholds have not been validated before (bug)

Reply:
I don't understand your comment. First of all, please put screenshots in the PR description and not into the comments. I cannot search through the whole history. Secondly, I would like to consistently two decimal points. We previously had an overall coverage between 0.88 and 0.92 and I think 0.8 is too low. If 0.89 is too high, let's go for 0.88 or 0.87, but please let's use two decimal points

Reply:
The relevant logs are linked in the PR description with an exact search term mentioned.

I got baited by the error message mentioning `0.8` when temporarily specifying `1.0`, despite it actually being `0.88`. Adapted the thresholds